META.d/_summary.yml

```
@@ -0,0 +1,11 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: MPL-2.0
+
+schema: 1.1
+partition: tf-ecosystem
+category: library
+
+summary:
+  owner: team-tf-editor-experience
```

```suggestion
  owner: team-tf-devex
```

We don't have a shared team ATM, but this matches the CODEOWNERS: https://heimdall.hashicorp.services/site/teams/team-tf-devex

---

I was following the ticket, but will definitely cede to the registered owner!